electrumx/server/http_session.py

```diff
@@ -110,6 +111,7 @@ def __init__(self, session_mgr, db, mempool, peer_mgr, kind):
         self.sv_seen = False
         self.MAX_CHUNK_SIZE = 2016
         self.hashX_subs = {}
+        self.op_list = {"mint-dft": 1, "dmint": 2, "dft": 3, "transfer": 4, "burn": 5, "invalid": 6, "split": 7}
```

Comment:
Can we also have: ... ?

- `mod`
- `seal`
- `mint-ft`
- `mint-nft`
-

Reply:
Should `burn` actually just be a status under the `transfer`?

Actually I have a question... how would you handle if multiple operation in the same tx?

For example if there is a transfer and a mint-dft and a burn all in the same tx?

Reply:
One tx just have one operation, if  a transfer and a mint-dft and a burn, it will be burn

Reply:
Yes, We can have all the operation, but need more test to cover all the situation

Reply:
Technically any operation can have a burn. Because a user could do `seal`, `mod`, `mint-ft`, `mint-dft`, `mint-nft`, etc, etc... and have a burn operation, including `split` and `splat` too! So therefore I think the "is_burn" should be a property of the struct.

Also, ideally we should have a struct like this:

```
{
is_burned: true,
burned_fts: {
  "123456...i0": <burn_amount>
}
}

```

The blueprint builder already returns the burned fts and we can then display it in the struct